### PR TITLE
Drop unconfineduser SELinux module.

### PR DESCRIPTION
**Pull request by sshimko**
We unintentionally left the unconfineduser
SELinux module enabled. Nothing was really
using it, as toor was mapped to either staff
or toor depending on the
CONFIG_BUILD_UNCONFINED_TOOR flag. Disable
unconfineduser in modules.conf.

Side-effect, build toor as an separate RPM
and conditionally add it to the ks packages
list when the flag mentioned above is set to
"y".

### CODE REVIEW

File: packages/selinux-policy/Makefile [hunk 1]
```diff
@@ -108,9 +108,9 @@ ifneq ($(CONFIG_BUILD_ENFORCING_MODE),n)
 else
 	ENFORCING_MODE := permissive
 endif
-#ifneq ($(CONFIG_BUILD_UNCONFINED_TOOR),n)
-#	ENABLE_MODULES += toor
-#endif
+ifneq ($(CONFIG_BUILD_UNCONFINED_TOOR),n)
+	SEPARATE_PKGS += toor
```
Comment (marshalljmiller) [inline packages/selinux-policy/Makefile]: I think the toor module should always be built as a separate package

Comment (sshimko) [reply packages/selinux-policy/Makefile]: Do you mean it should always be present, but not inserted into the store?

Or do you mean, CONFIG_BUILD_UNCONFINED_TOOR is "y", it should be put in a separate RPM?

If it is the second one, SEPARATE_PKGS should override whatever is in modules.conf, and then place the PP in a separate RPM.

The paptch dropped the commented out piece that only enabled it, and didn't place it in a separate RPM.

Comment (marshalljmiller) [reply packages/selinux-policy/Makefile]: It should always be built and included in its own package (IOW, always present in SEPARATE_PKGS).  The kickstart should decide if it gets installed or not.

Comment (sshimko) [reply packages/selinux-policy/Makefile]: That makes sense that removes the logic in the policy RPM Makefile and leaves it in the ks. I'll update this.

**Pull request merged**